.env

```diff
@@ -0,0 +1,21 @@
+NEXT_PUBLIC_APP_TITLE=Create Auction House ☼☽
```

Comment:
I like the comments already here: can we add more perhaps and organize into sections?